Automattic/newspack-theme
# Add basic WooCommerce support

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds very basic WooCommerce styles and support to the theme, based off of the styles and support baked into the WC plugin for Twenty Nineteen, as well as some visual tweaks for extensions like Name Your Price and One Page Checkout. 

This PR is not intended to be the final WooCommerce support. It is meant to clean up some more obvious visual issues with the plugin for the short-term; full support will need additional work and more in-depth testing. 

### How to test the changes in this Pull Request:

1. Follow the steps outlined here to create a product: pamTN9-aY-p2.
2. Publish and view the product; note visual issues like the tab styles not matching the styles, and weird font sizing.
3. Apply the PR; confirm that the visual issues are largely resolved.
